src/containers/shared/components/Streams.jsx

```diff
@@ -118,7 +118,7 @@ class Streams extends Component {
   componentDidMount() {
     this.connect()
     this.updateNegativeUNL()
-    if (process.env.REACT_APP_ENVIRONMENT !== 'sidechain') {
+    if (process.env.REACT_APP_ENVIRONMENT !== 'custom') {
```

Comment:
Should we hide these metric sections from the homepage if we don't retrieve the data for them?

A change for another PR.

Reply:
We don't retrieve data from the server, it's generated via the data the Explorer stores locally. See line 502